src/multisig/utils.ts

```diff
@@ -122,6 +122,19 @@ export const getContractNetworks = (chainId: number): ContractNetworksConfig =>
         simulateTxAccessorAddress: "0xe7A7552B0Cff45E837e1422b479d2dEDE748f571",
       },
     },
+    // safe v1.4.1
+    opmainnet: {
```

Comment:
Only testnet addresses are stored here. I think we should remove it.

Reply:
fixed, https://github.com/VenusProtocol/vips/pull/386/commits/213713fb4032b9dd9b7e580c0ec8d183a5a3ef8b